nav2_mppi_controller/src/optimizer.cpp

```diff
@@ -77,6 +77,10 @@ void Optimizer::getParams()
   getParam(s.base_constraints.vx_min, "vx_min", -0.35f);
   getParam(s.base_constraints.vy, "vy_max", 0.5f);
   getParam(s.base_constraints.wz, "wz_max", 1.9f);
+  getParam(s.base_constraints.ax_max, "ax_max", 2.0f);
```

Comment:
We should set these in the default Nav2 config file now that MPPI is default. Probably set it to something high so that it doesn't impact performance but can be bounded by the user when they want to.

Reply:
We also need to add this to our migration guide that we added acceleration constraints to MPPI + to the MPPI's configuration guide on docs.nav2.org

Reply:
Do me a favor as well and check on the signs of the acceleration and at least log a warning if the acceleration signs are wrong (i.e. ax_min being positive, for example) and set them to be rationally correct